### PR TITLE
Miscellaneous fixes (obsolete db close, API error codes, heritability report tweaks)

### PR DESCRIPTION
**Pull request by vlagrassa**
A few minor tweaks, bundled together:

- Update a few API error codes, to be more semantically accurate
- Remove obsolete code to explicitly close db on app context teardown
  - This is handled by Flask SQLAlchemy
  - I originally wrote these functions trying to debug the stale GCP connection issue (see #387 for the actual fix)
- Heritability report:
  - Update graph size on fullscreen
  - Order AssayNumber numerically (1, 2 ... 9, 10 ...) instead of alphabetically (1, 10, 11 ... 2, 3 ...)

### CODE REVIEW

File: src/modules/site-v2/templates/tools/heritability_calculator/result.html [hunk 1]
```diff
@@ -209,14 +210,26 @@
...
-        width = document.getElementById("svgchartarea").offsetWidth - 100; /* 100px is from the chart legend*/
-        height = 9 * width / 16;
-        console.log(width)
-        var chart = d3.exploding_boxplot(df, {y:'Value', group:'Strain', color:'AssayNumber', label:'TraitName', pal:cpal})
-                      .width(width)
-                      .height(height);
-        document.getElementById("svgchartarea").innerHTML = "";
-        chart('#svgchartarea');
+        // Create & save a function to compute the chart size & re-render it
+        renderPlot = function() {
```
Comment (r-vieira) [inline src/modules/site-v2/templates/tools/heritability_calculator/result.html:214]: when declaring JS vars in new code, prefer `let` and `const`. For lambda-type functions, use `const`

Comment (r-vieira) [reply src/modules/site-v2/templates/tools/heritability_calculator/result.html:214]: consider `const renderPlot = ...`

File: src/modules/site-v2/templates/tools/heritability_calculator/result.html [hunk 2]
```diff
@@ -209,14 +210,26 @@
...
+          // Clear the current chart area
+          // Otherwise chart can't shrink - width computation will always be based on existing size
+          document.getElementById("svgchartarea").innerHTML = "";
+
+          // Take up as much horizontal space as available, keeping a 16:9 ratio
+          width = document.getElementById("svgchartarea").offsetWidth - 100; /* 100px is from the chart legend*/
+          height = 9 * width / 16;
+
+          // Create the chart and render in the page
+          var chart = d3.exploding_boxplot(df, {y:'Value', group:'Strain', color:'AssayNumber', label:'TraitName', pal:cpal})
```
Comment (r-vieira) [inline src/modules/site-v2/templates/tools/heritability_calculator/result.html:225]: prefer `const chart = ...`

**Review by r-vieira — APPROVED** [review]
looks good

File: src/modules/site-v2/templates/tools/heritability_calculator/result.html (hunk 2 above)
Comment (vlagrassa) [reply src/modules/site-v2/templates/tools/heritability_calculator/result.html:225]: Fixed in 94bdca02fdd6f97a4d5ba2b37220d4be65abe2a8

File: src/modules/site-v2/templates/tools/heritability_calculator/result.html (hunk 1 above)
Comment (vlagrassa) [reply src/modules/site-v2/templates/tools/heritability_calculator/result.html:214]: In this particular case, I'm actually re-assigning the value of `renderPlot` (declared on line 207) to make it available outside the inner function context.  (Not the best solution, but it involved the least code restructuring.)

For the top-level declaration of `renderPlot`, updated to `let` in 94bdca02fdd6f97a4d5ba2b37220d4be65abe2a8

**Pull request merged**